definitions/variable/tag_plant.yaml

```
@@ -0,0 +1,13 @@
+- Plant:
+    - Power Plant:
```

Maybe be more explicit
```suggestion
    - Thermal Power Plant:
```

---

"Power plant" is meant to refer to anything that (exclusively) generates electricity. How about "electricity generation unit"?

---

This tag seems to be used for the variable `Transformation Input|{Plant}` and `Transformation Input|{Plant}` - but several of the "plants" don't have an input. So I'm not convinced that this is a useful approach.

In the openENTRANCE and ECEMF projects, we instead used the following approach:
- use `Secondary Energy|...` for the output of any technology along the energy supply chain (in line with the usage of this variable in AR6 and other projects
- use `Conversion Input|...` to denote the quantities that are inputs along the energy supply chain 
See [here](https://github.com/openENTRANCE/openentrance/blob/6d6ff3c58b940c5a270cdf2ee28a48c8f5fa8461/definitions/variable/energy/energy-secondary.yaml#L183)

---

Not sure what you mean by 
> several of the "plants" have no input
As far as I understand, `Transformation Input` is the TIMES-compatible wording for `Conversion Input`. As we sometimes have more than one technology to convert `Transformation Input` to `Secondary Input`, we'd like to include a `Plant`-dimension.

---

I meant: You suggest to have "Reservoir Storage" in the list of plants, but this technology does not have any (energy) input that is reported in this energy-flow perspective - so I'm not convinced that this structure makes sense.

It's a good idea to report energy-inputs with the power-plant-type-dimension explicitly, but the current list of plants does not really give sufficient disaggregation (e.g. no distinction between CCGT vs. OCGT power plants). So I would rather follow the structure
`Conversion Input|Gas|Electricity|Closed-Cycle` to keep track of the amount of natural gas used in that CCGT plants.